### DISCUSSION TITLE
Filter out files for `grep_string`

### DISCUSSION THREAD

**Discussion by thuan1412** [nvim-telescope/telescope.nvim#1445] [Q&A] [answered]
How do you filter out files when you use `grep_string`? 
I just want to `grep_string` in files that match specific patterns. I am aware that we can put some options to the `grep_string` function, but I think that is not a convenient way.

**Comment by Conni2461** [comment, marked as answer]
You can filter out with [fzf-native](https://github.com/nvim-telescope/telescope-fzf-native.nvim) as a sorter
